### DISCUSSION TITLE
Windows 11 UI concept

### DISCUSSION THREAD

**Discussion by inkwadra** [transmission/transmission#6826] [Ideas]
I understand that it’s inappropriate to post this on GitHub, but for some reason, I couldn’t register on the forum, so I decided to share my work here. Recently, I finished the Transmission design concept for Windows, and I am well aware that implementing it is extremely difficult and time-consuming. However, if it serves as inspiration in the future, I would be very flattered.

![Transmission](https://github.com/transmission/transmission/assets/42640/ce4d89e9-96a6-4172-bc58-9d0ed9f14213)

[Dribbble Shot](https://dribbble.com/shots/24104923-Transmission-Redesign)

**Comment by Pentaphon** [comment]
Looks good! I would definitely like to see most of these design elements for Windows but I'm not sure if people need that many sorting icons which could instead be put into a combobox. If you are able to contribute to the Qt client, please take a look at https://github.com/transmission/transmission/pull/4376

**Comment by killemov** [comment]
> I understand that it’s inappropriate to post this on GitHub,

You can't post it on GitHub, but can share it on GitHub? You meant as a separate repo?

Anyway, this does look nice and clean. How/where do you get the related mime-type icon?

**Reply by inkwadra** [reply]
I use PowerShell for extract icons from DLL
```powershell
function Get-AllIcons {
    $id = 0

    while ($true) {
        $icon = [System.Drawing.Icon]::ExtractIcon("C:\Windows\System32\imageres.dll", $id, $false)
        if ($icon -ne $null) {
            $icon.ToBitmap().Save("C:\Icons\$id.png")
            $id++
        }
        else {
            break
        }
    }
}
```
